test/Test-EndToEnd.js

```diff
@@ -216,6 +216,41 @@ describe('EndToEnd', () => {
         });
     });
 
+    describe('FileStore with relativeLocation', () => {
+        let file_id;
+        let deferred_file_id;
```

Comment:
`deferred_file_id` is just for the End To End test on deferring the file location. Do you need to store `file_id` and  `deferred_file_id` in a variable?

Reply:
Ah, no just remains from the whole test i copied before, i'll remove it.